api-reference/10 UI Widgets/dxScheduler/3 Methods/scrollTo(date_group_allDay).md

```diff
@@ -3,16 +3,24 @@ id: dxScheduler.scrollTo(date, group, allDay)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Scrolls the current view to a specified position. Available for all views except *"agenda"*. Requires the [height](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#height) property to be set.
 
 ##### param(date): Date
-<!-- Description goes here -->
+A date and time to scroll to.
 
 ##### param(group): Object | undefined
-<!-- Description goes here -->
+An appointment group (optional).
 
 ##### param(allDay): Boolean | undefined
-<!-- Description goes here -->
+If **true**, scrolls to the all-day panel of the specified group. Applies only if the all-day panel is [visible](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#showAllDayPanel).
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+[note] If you need to navigate to a date outside of the current view, use the [currentDate](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#currentDate) property instead. 
+
+#####See Also#####
+#include common-link-callmethods
+- [Group Appointments by Resources](/Documentation/Guide/Widgets/Scheduler/Resources/Group_Appointments_by_Resources/)
+- [currentView](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#currentView)
+- [startDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#startDayHour)
+- [endDayHour](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#endDayHour)
```

Comment:
How are these properties connected with the method? Let's give links to the scrolling object and the crossScrollingEnabled property instead.